resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/submit/v2/Client.scala

```diff
@@ -227,8 +235,10 @@ private[spark] class Client(
       resolvedLocalClassPath.foreach { classPathEntry =>
         require(Option(URI.create(classPathEntry).getScheme).isEmpty)
       }
-      val resolvedDriverJavaOpts = resolvedSparkConf.getAll.map { case (confKey, confValue) =>
-          s"-D$confKey=$confValue"
+      sparkConfExecutorsFetchRemoteDeps.set(
+        EXECUTOR_RESOLVED_MOUNTED_CLASSPATH, resolvedLocalClassPath)
```

Comment:
Or we can just add `jarsDownloadPath/*` to the classpath. Everywhere in Spark the exact jar files are enumerated, but since we're guaranteeing that the download path is an empty directory volume mount and is strictly being used for jars, it might make sense to just reference the whole directory in this context.